cli/flox-rust-sdk/src/models/manifest/typed.rs

```diff
@@ -993,7 +993,7 @@ impl SkipSerializing for Include {
 #[serde(deny_unknown_fields)]
 #[serde(
     untagged,
-    expecting = "Must specify included environment as { dir = <dir>, [name = <name>] }"
+    expecting = "expected `dir`, `remote`, or `name` for specifying included environment"
```

Comment:
`name` by itself isn't enough, despite the sentence saying so.
```suggestion
    expecting = "expected { dir = <dir>, [name = <name>] } OR { remote = <owner/name>, [name = <name>] }"
```